handbook/engineering/engineering-management.md

```diff
@@ -49,20 +49,43 @@ Here are some books that multiple people on the team have read and recommend. If
+Teammates who have reached the [IC3 level](career-development/framework.md#ic3) and have a track record of good performance at Sourcegraph, having been employed for at least six months, are eligible for a manager in training trial.
+The teammate should make their desire clear to their Engineering Manager so that when a relevant opportunity becomes available, they may be considered.
 
-This process aims to have a determination made by the interim manager's manager about the promotion to a permanent manager role after the agreed success criteria have been satisfied. Generally, we aim to have the interim period not exceed four months before turning into a full-time position.
+Opportunities for a manager in training trial can come about through a current Engineering Manager position becoming vacant, an existing team splitting into two, or an Engineering Manager going on long-term PTO. 
+When an opportunity arises, the teammate's manager can nominate them for a manager in training trial to the Director of Engineering for the relevant org.
+
+There are some considerations to keep in mind to ensure that the teammate is set up for success:
+- The teammate's existing team should be in a position for them not to be involved as an IC for the whole duration of the trial period. Managers in training should not fulfill a dual role.
+- The team to be managed should comply with our [minimum viable team](eng_org.md#minimum-viable-team) criteria.
+- The team to be managed should have a dedicated and established Product Manager assigned to it.
+
+### Manager in training trial period
+
+During the trial period, the manager in training will assume full Engineering Manager responsibility as outlined in the [Engineering Manager responsibilities](roles.md#engineering-manager).
+
+The teammate's manager or another nominated manager will become their sponsoring manager for the duration of the trial period. The sponsoring manager must commit to providing extra support to the manager in training during the trial period, which will last 12 weeks. 
+The trial period may be extended if agreed to by the manager in training and the sponsoring manager. However, the trial period should not be extended to more than 16 weeks.
+
+The sponsoring manager will give the manager in training active coaching and feedback during weekly 1-1s and conduct bi-weekly 1-1s with the manager in training's direct reports to collect feedback and set them up for success.
+Halfway through their trial period, the manager in training will receive an interim performance evaluation to help them identify strengths and shortcomings as an engineering manager and allow them to adjust if needed.
+
+The trial period can be canceled by the manager in training or the sponsoring manager at any time if it is not working out.
+
+### Promotion to Engineering Manager
```

Comment:
If the IC was level IC5 (or higher) before this wouldn't be a promotion but a lateral move. Should this perhaps not use the word "promotion"? Or would in that case the process below not be followed (no "promotion request" need to be filed)?

I think it's important to communicate that becoming an EM is first and foremost a role change and not _necessarily_ a promotion, since we do think of IC5 as equivalent in impact to an EM.

Reply:
I think it should still require a "promotion" request to be completed since it is a formal change. However, you call out an important point that the wording should reflect that it's a role change, not a promotion. I'll update the wording now.